tms/clickup/serializer.py

```diff
@@ -22,3 +23,17 @@ def create(self, validated_data):
         validated_data.pop('password_confirm')
         user = super().create(validated_data)
         return user
+
+
+class LoginSerializer(serializers.Serializer):
+    username = serializers.CharField()
+    password = serializers.CharField()
+
+    def create(self, validated_data):
+        username = validated_data["username"]
+        password = validated_data["password"]
+        user = authenticate(username=username, password=password)
+        if not user:
+            raise serializers.ValidationError(constant.LOGIN_ERROR_MESSAGE)
+        token, _ = Token.objects.get_or_create(user=user)
```

Comment:
do not genrate token here , if user is authenticate then return user else error that's it.